tests/gds_kernel_latency.c

```
@@ -730,11 +792,22 @@ int main(int argc, char *argv[])
 			peersync = !peersync;
                         printf("INFO: switching PeerSync %s\n", peersync?"ON":"OFF");
 			break;
+			
+		case 'K':
+			gds_qpt = (int) strtol(optarg, NULL, 0);
+                        printf("INFO: GDS_QPT %s\n", (gds_qpt == 1) ? "UD":"RC");
+			gds_qpt = (gds_qpt == 1) ? IBV_QPT_UD : IBV_QPT_RC;
```

can you return an error if gds_qpt is != (1,2) ?